tests/30rooms/14forget.pl

```diff
@@ -0,0 +1,298 @@
+use List::Util qw( any none );
+
+test "Forgotten room messages cannot be paginated",
+   requires => [ local_user_and_room_fixtures(), local_user_fixture() ],
+
+   do => sub {
+      my ( $creator, $room_id, $user ) = @_;
+
+      matrix_join_room( $user, $room_id )
+      ->then( sub {
+         matrix_send_room_text_message( $creator, $room_id, body => "sup" )
+      })->then( sub {
+         do_request_json_for( $user,
```

Comment:
FYI, these could become oneline calls to `matrix_get_room_messages()` once PR #84 lands on develop.